### ISSUE TITLE
Installation script fails on CentOS 7 due to old Wget

### ISSUE THREAD

**Issue by mcejp** [laurent22/joplin#4733]
## Environment

Joplin version: 0da3881aa6cddd465f6fba8ab8b0883b841cb83f (HEAD of dev at the moment of writing)
Platform: Linux
OS specifics: CERN CentOS 7. `cat /etc/redhat-release -> CentOS Linux release 7.9.2009 (Core)`

## Steps to reproduce

1. Run the recommended installation script `wget -O - https://raw.githubusercontent.com/laurent22/joplin/dev/Joplin_install_and_update.sh | bash`
2. Error `wget: unrecognized option '--show-progress'` when executing the downloaded script.

## Describe what you expected to happen

Joplin is installed.

## Logfile

```
$ wget -O - https://raw.githubusercontent.com/laurent22/joplin/dev/Joplin_install_and_update.sh | bash
--2021-03-26 09:37:18--  https://raw.githubusercontent.com/laurent22/joplin/dev/Joplin_install_and_update.sh
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.111.133, 185.199.110.133, 185.199.109.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.111.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 8686 (8.5K) [text/plain]
Saving to: ‘STDOUT’

100%[==================================================================================================================>] 8,686       --.-K/s   in 0s      

2021-03-26 09:37:18 (33.9 MB/s) - written to stdout [8686/8686]


     _             _ _       
    | | ___  _ __ | (_)_ __  
 _  | |/ _ \| '_ \| | | '_ \ 
| |_| | (_) | |_) | | | | | |
 \___/ \___/| .__/|_|_|_| |_|
            |_|

Linux Installer and Updater

Checking architecture...
The latest version is 1.7.11, but you have no version installed.
Downloading Joplin...
wget: unrecognized option '--show-progress'
Usage: wget [OPTION]... [URL]...

Try `wget --help' for more options.
```

## Discussion

The work-around is easy: download the installer to a file and manually remove the 2 occurences of `--show-progress`.

CentOS 7 ships wget 1.14, which is like 10 years old. As such, it would probably make sense to leave this as WONTFIX.

**Comment by laurent22** [comment]
Could you perhaps create a pull request that removes this "show-progress" flag from the script?

**Comment by mcejp** [comment]
Absolutely, but if I understand well, it will lead to either
- verbose output like in [this StackOverflow question](https://stackoverflow.com/q/4686464), or
- no progress indication at all (if we keep `-q`)

Is it worth it?

**Comment by laurent22** [comment]
I think verbose output is fine as it's better if we can see that something's happening, especially if for some reason it's not working.

**Comment by tessus** [comment]
We could also try to use curl instead, but we might run into the same "not supported" issue as well. I don't have a CentOS 7 to test. Or is it more likely that people have wget installed but not curl? It was just an idea...

**Comment by tessus** [comment]
fixed in ea65346f04669085128ae8129b208d0ca5d0f671
